What "lib" values are compatible with Deno?

Hi,

in my tsconfig.json:

```json
"compilerOptions": {
  "lib": ["esnext", "dom", "deno.ns"]
}
```

...and with Deno 0.35.0:

```sh
error TS2488: Type 'Headers' must have a '[Symbol.iterator]()' method that returns an iterator.

► https://deno.land/std/http/io.ts:135:24

135   for (const [k, v] of result) {
                           ~~~~~~

error TS2488: Type 'Headers' must have a '[Symbol.iterator]()' method that returns an iterator.

► https://deno.land/std/http/io.ts:206:30

206   for (const [key, value] of trailers) {
                                 ~~~~~~~~

error TS2488: Type 'Headers' must have a '[Symbol.iterator]()' method that returns an iterator.

► https://deno.land/std/http/io.ts:260:30

260   for (const [key, value] of headers) {
```

@kitsonk I am getting about 50 of these signature errors using your compileroptions:

> error TS2374: Duplicate string index signature.
> ► $asset$/lib.dom.iterable.d.ts:6944:5
> 6944     [name: string]: any;

--- a/deno_typescript/lib.rs
+++ b/deno_typescript/lib.rs
@@ -251,7 +251,7 @@ pub fn get_asset(name: &str) -> Option<&'static str> {
     "bootstrap.ts" => Some("console.log(\"hello deno\");"),
     "typescript.d.ts" => inc!("typescript.d.ts"),
     "lib.dom.d.ts" => inc!("lib.dom.d.ts"),
-    "lib.dom.iterable.d.ts" => inc!("lib.dom.d.ts"),
+    "lib.dom.iterable.d.ts" => inc!("lib.dom.iterable.d.ts"),
     "lib.es5.d.ts" => inc!("lib.es5.d.ts"),
     "lib.es6.d.ts" => inc!("lib.es6.d.ts"),
     "lib.esnext.d.ts" => inc!("lib.esnext.d.ts"),

